src/robusta/core/model/base_params.py

```diff
@@ -171,6 +171,33 @@ class ConversationType(str, Enum):
     ISSUE = "issue"
 
 
+class HolmesIssueChatParamsContext(BaseModel):
+    """
+    :var investigation_result: HolmesInvestigationResult object that contains investigation saved to Evidence table by frontend for the issue.
+    :var issue_type: aggregation key of the issue
+    :var robusta_issue_id: id of the issue
+    """
+
+    investigation_result: HolmesInvestigationResult
```

Comment:
This class is duplicated with `HolmesConversationIssueContext`

Why do we need both?
Can we use the same class, or make one of them extend the other?